test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -594,3 +595,35 @@ func (sr *SnapshotResource) CleanupResource() error {
 	}
 	return utilerrors.NewAggregate(cleanupErrs)
 }
+
+// CheckDeleteSnapshotSecrets checks if delete snapshot secrets annotation is applied to the VolumeSnapshotContent.
+func CheckDeleteSnapshotSecrets(cs clientset.Interface, annotations interface{}) error {
```

Comment:
Since this function is only going to be used by csi_mock_volume.go, can you move over there?